commonknowledge/mapped
# feat: make pledge page work out of the box after project setup

## Description
Updates the seed command so that the pledge feature works out of the box.

## Motivation and Context
https://linear.app/commonknowledge/issue/MAP-578/the-test-hub-pledge-in-development-is-broken

## How Can It Be Tested?
Either set up a fresh install of Mapped and run through the README setup instructions, or run `python manage.py seed` inside the VSCode dev container.

Then go to http://hub.localhost:3000, navigate to the Map, search for a postcode, then click "Pledge". The form will appear. Fill in the form and submit it. Check the data has come through in the AirTable here: https://airtable.com/appeIkNt2Lp1SkAdD/tblzFqsdybJRxAbb2/viwzXnj1NM2kI8b03?blocks=hide
